lib/widget/rich_menu.dart

```diff
@@ -50,18 +50,11 @@ class _RichMenuState extends State<RichMenu> {
                   ),
                 ],
               ),
-              FlatButton(
-                padding: EdgeInsets.only(top: 0),
-                onPressed: () {
-                  Navigator.pushNamed(context, '/evaluation_page');
-                },
+              TextButton(
+                style: TextButton.styleFrom(backgroundColor: Color(0xFFFCECBC)),
                 child: Container(
                   height: 100,
                   width: MediaQuery.of(context).size.width,
-                  decoration: BoxDecoration(
-                    borderRadius: BorderRadius.circular(0),
-                    color: Color(0xFFFCECBC),
-                  ),
```

Comment:
This could be design change? or the migration does not complete here?

Reply:
<img width="346" alt="Screen Shot 2564-07-19 at 19 54 50" src="https://user-images.githubusercontent.com/57941667/126162799-367df2be-e177-47e8-bedc-65db2726cd9e.png">

The UI result remains the same, but the richmanu is migrated with textbutton, so the code has changed a bit.